src/components/LearnerActivityTable/index.jsx

```diff
@@ -1,152 +1,200 @@
-      },
-      {
-        label: intl.formatMessage({
-          id: 'admin.portal.lpr.learner.activity.table.course_start_date.column.heading',
-          defaultMessage: 'Start Date',
-          description: 'Column heading for the course start date column in the learner activity table',
-        }),
-        key: 'course_start_date',
-        columnSortable: true,
-      },
-      {
-        label: intl.formatMessage({
-          id: 'admin.portal.lpr.learner.activity.table.course_end_date.column.heading',
-          defaultMessage: 'End Date',
-          description: 'Column heading for the course end date column in the learner activity table',
-        }),
-        key: 'course_end_date',
-        columnSortable: true,
-      },
-      {
-        label: intl.formatMessage({
-          id: 'admin.portal.lpr.learner.activity.table.passed_date.column.heading',
-          defaultMessage: 'Passed Date',
-          description: 'Column heading for the passed date column in the learner activity table',
-        }),
-        key: 'passed_date',
-        columnSortable: true,
-      },
-      {
-        label: intl.formatMessage({
-          id: 'admin.portal.lpr.learner.activity.table.current_grade.column.heading',
-          defaultMessage: 'Current Grade',
-          description: 'Column heading for the current grade column in the learner activity table',
-        }),
-        key: 'current_grade',
-        columnSortable: true,
-      },
-      {
-        label: intl.formatMessage({
-          id: 'admin.portal.lpr.learner.activity.table.progress_status.column.heading',
-          defaultMessage: 'Progress Status',
-          description: 'Column heading for the progress status column in the learner activity table',
-        }),
-        key: 'progress_status',
-        columnSortable: true,
-      },
-      {
-        label: intl.formatMessage({
-          id: 'admin.portal.lpr.learner.activity.table.enrollment_date.column.heading',
-          defaultMessage: 'Last Activity Date',
-          description: 'Column heading for the last activity date column in the learner activity table',
-        }),
-        key: 'last_activity_date',
-        columnSortable: true,
-      },
-    ];
-
-    if (activity !== 'active_past_week') {
-      return tableColumns.filter(column => column.key !== 'passed_date');
-    }
-    return tableColumns;
-  }
+import { formatPrice } from '../learner-credit-management/data';
+import { useTableData } from '../Admin/TableDataContext';
+import { updateUrlWithPageNumber } from './data/utils';
+
+const FilterStatus = (rest) => <DataTable.FilterStatus showFilteredFields={false} {...rest} />;
+
+const UserEmail = ({ row }) => (
+  <span data-hj-suppress>{row.original.userEmail}</span>
+);
+
+UserEmail.propTypes = {
+  row: PropTypes.shape({
+    original: PropTypes.shape({
+      userEmail: PropTypes.string,
+    }).isRequired,
+  }).isRequired,
+};
+
+const LearnerActivityTable = ({ id, enterpriseId, activity }) => {
```

Comment:
[suggestion] Given a unique id is available via the top level component rendering each individual table, (along with the custom search filter component, `AdminSearchForm`) in we can utilize this value to start prepending ur relevant filters and page information with the identifier with this value.

This will allow the URL navigation to display its corresponding table, and update the filter and page number with the correct information from the API.

Reply:
Are you suggesting adding tableId + pageNumber so that we can use a consistent structure for identifying table states across different tables while navigating?

Reply:
Yes exactly 👍🏽

Reply:
I have implemented this change and now URL will look like 
http://localhost:1991/test-enterprise/admin/learners/learners-inactive-week?learners-inactive-week_page=2